app/controllers/concerns/sufia/depositors_controller_behavior.rb

```diff
@@ -24,6 +24,7 @@ def create
         head :ok
       else
         grantor.can_receive_deposits_from << grantee
+        send_proxy_depositor_added_messages(grantor, grantee)
```

Comment:
Can you make this a subclass of EventJob and call it asynchronously (`perform_later`)? For example: https://github.com/projecthydra/sufia/blob/master/app/controllers/concerns/sufia/users_controller_behavior.rb#L91